crud/XlsParserAction.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * @author Patryk Radziszewski <pradziszewski@netis.pl>
+ * @link http://netis.pl/
+ * @copyright Copyright (c) 2015 Netis Sp. z o. o.
+ */
+
+namespace nineinchnick\sync\crud;
+
+use nineinchnick\sync\models\parser\BaseXlsParser;
+use Yii;
+
+class XlsParserAction extends CsvParserAction
+{
+
+    public $parser = 'app\models\OrderXlsParser';
+    public $scenario = BaseXlsParser::SCENARIO_XLS_PARSER;
+    public $modelClass = 'nineinchnick\sync\models\parser\BaseXlsParser';
+
+    /**
+     * @inheritdoc
+     */
+    protected function initModel($id)
+    {
+        $model = parent::initModel($id);
+        if (!Yii::$app->getRequest()->getIsPost() && is_null($model->columnsOrder)) {
+            $model->columnsOrder = json_encode($this->getDefaultColumnsOrder());
+        }
+        return $model;
+    }
+
+    /**
+     * Static map of columns order
+     * @return array
+     */
+    public function getDefaultColumnsOrder()
+    {
+        return [
+            'entrydt' => Yii::t('nineinchnick/sync/models', 'Entry Date'),
```

Comment:
@pradziszewski123  remove those columns from this extension, they're specific to a private project.